### ISSUE TITLE
Error loading CTI locally 

### ISSUE THREAD

**Issue by GPHofficial** [mitre-attack/mitreattack-python#14]
To replicate:

```
from mitreattack.navlayers import MatrixGen
MatrixGen(source='local', local='cti/enterprise-attack/enterprise-attack.json')
```

matrix_gen.py, [line 123](
https://github.com/mitre-attack/mitreattack-python/blob/53dd269e532e58a152be1ffe336da5d528e8fae9/mitreattack/navlayers/exporters/matrix_gen.py#L123):

`self.collections['enterprise'] = hd.load_from_file(local)`
`hd.load_from_file(local)` returns None

`self.collections['enterprise'] = hd`
should work as per normal

https://stix2.readthedocs.io/en/latest/api/datastore/stix2.datastore.memory.html#stix2.datastore.memory.MemoryStore.load_from_file

**Comment by ArtificialErmine** [comment]
Thanks for pointing this out! I'll look into it and see what I can find.

**Comment by ArtificialErmine** [comment]
I've made a patch and put it in the pipeline to be released. Thanks for bringing this to our attention!

**Comment by ArtificialErmine** [comment]
This has been patched as of v1.2.1. Thanks for reporting this, and don't hesitate to let us know about any other issues you may encounter.
